fe1-web/src/features/wallet/screens/__tests__/WalletHome.test.tsx

```diff
@@ -27,6 +26,10 @@ jest.mock('core/platform/crypto/browser');
 jest.mock('features/wallet/objects/Wallet');
 jest.mock('core/components/QRCode.tsx', () => 'qrcode');
 
+// disable animations
+jest.useFakeTimers('modern');
+jest.setSystemTime(new Date(1620255600000)); // 5 May 2021
```

Comment:
Because it makes the snapshot tests always return different things as the animation has made different progress. (The accordion items that open by default are open with an animation)